activation/nipost.go

```diff
@@ -100,7 +101,7 @@ func NewNIPostBuilder(
 		layerClock:     layerClock,
 		postStates:     NewPostStates(lg),
 		validator:      validator,
-		identityStates: NewIdentityStateStorage(),
+		identityStates: identity.NewIdentityStateStorage(),
```

Comment:
Why does the `NIPostBuilder` have its own `identity.StateStorage`? Shouldn't this be injected from `node.go` like it is for the `miner.RemoteBuilder`?

Reply:
It is set by using `NipostbuilderWithIdentityStates`. As this is PoC, I don't want to change all tests to supply identity storage everywhere.